types/twine-sugarcube/extensions.d.ts

```diff
@@ -27,6 +27,17 @@ declare global {
          */
         count(needle: T, position?: number): number;
 
+        /**
+         * Returns the number of times that members within the array pass the test implemented by the given predicate function.
+         * @param predicate The function used to test each member. It is called with three arguments:
```

Comment:
@ezsh We've lost the arguments to the predicate here.  Yes, they're standard for an array predicate and they're actually included within the definition itself, but it would be nice to have them documented.  If that's not feasible, then no biggie I guess.

Reply:
I think their names give enough information except the last one, `array`, but in the TS notation its type (`this`) resolves doubts. But if you want to provide textual descriptions, no problem to do that, of course.

Reply:
Ah, you probably meant abrupt ending after a colon. I corrected the comment and fixed mistake for `deleteWith()`.

Reply:
Nope, I did not...